Sources/BraintreeCore/BTShopperInsightsClient.swift

```diff
@@ -0,0 +1,27 @@
+import Foundation
+
+///  Use `BTShopperInsightsClient` to optimize your checkout experience by prioritizing the customer’s preferred payment methods in your UI.
+///  By customizing each customer’s checkout experience, you can improve conversion, increase sales/repeat buys and boost user retention/loyalty.
+/// - Note: This feature is in beta. It's public API may change or be removed in future releases.
+@objcMembers class BTShopperInsightsClient: NSObject {
```

Comment:
These are great questions - I honestly am very down to not add Objective-C support to this feature. We could always add it later if we see demand for it. Also worth noting - this feature will go out as an "alpha" so we have the room to iterate on it, break the API, etc.

Done in 0884b7835516edddce143676ef302679ce747831. Will loop @jaxdesmarais in on this one though to confirm.

Reply:
> Actually I see now that this feature lives in core - what is the reason for not having a separate module for this?

We could make this a separate module. I was approaching this from a path of least resistance approach for now as we target "alpha". We might save us the headache from creating a documenting an entire new module, if the feature doesn't get adopted. And should be relatively simple to move the files into a module if we do want to go that route later on. Open to folks thoughts, though!

Reply:
Let's do it!

Reply:
Should we also drop the BT prefix? Or do we want to leave that consistent until v7?

Reply:
Another great question - maybe let's leave the prefix until we're ready to do that overhaul holistically?

Reply:
That makes sense to me! I agree it could be confusing to have some things prefixed and some not so the consistency would be great then we can say goodbye to it in v7. 🚀